openvm/tests/apc_builder_pseudo_instructions.rs

```diff
@@ -177,7 +177,8 @@ fn ret() {
 }
 
 #[test]
-fn add_constant() {
+fn load_immediate() {
+    // [x48] = [x0] + 216 = 216
     let program = [add(48, 0, 216, 0)];
-    assert_machine_output(program.to_vec(), "add_constant");
+    assert_machine_output(program.to_vec(), "load_immediate");
```

Comment:
This name seemed more descriptive. Loading an immediate is also a common [RISC-V pseudo instruction](https://michaeljclark.github.io/asm.html), though it can be implemented in different ways.